contracts/tournament/Tournament.cairo

```diff
@@ -61,6 +76,21 @@ end
 func max_ships_per_tournament_() -> (res : felt):
 end
 
+# Size of the grid
```

Comment:
I have the feeling that `grid_size_`, `turn_count_` and `max_dust_` could be merged in a single felt given their realistic maximum size. It would make the code less readable for sure but it would save storage space. 
It depends on what we want to emphasise regarding this code. 
Simply put: do we want to promote best practices for production code or do we want to focus on learning concepts ?
If we want the former then storage optimization should be a priority. If we want more the latter then code readability should be more important.

Reply:
I have the feeling the code clarity drop is not worth the size gain in this case